src/main/java/org/testobject/rest/api/appium/common/data/DataCenterSuite.java

```diff
@@ -0,0 +1,43 @@
+package org.testobject.rest.api.appium.common.data;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.net.URL;
+import java.util.Set;
+
+public class DataCenterSuite {
```

Comment:
Please always put JsonIgnore(ignoreUnknown) . It is a good idea to always put that into our public APIs